### PR TITLE
Update registration functionality to accept extrinsics weight

### PR DESCRIPTION
**Pull request by pawanbisht62**
This PR pertains to receiving extrinsic's benchmarked weights of tellor pallet.

As each parachain will be having different weights based on the pallet configuration, that's why we are keeping weights of each parachain.

Resolves: [issue](https://github.com/tellor-io/tellor-pallet/issues/90)

### CODE REVIEW

File: src/ParachainRegistry.sol [hunk 1]
```diff
@@ -13,8 +13,24 @@ interface IRegistry {
         bytes palletInstance;
         uint256 weightToFee;
         XcmTransactorV2.Multilocation feeLocation;
+        uint64 reportStakeDeposited;
```
Comment (evilrobot-01) [inline src/ParachainRegistry.sol:16]: Why not just use the weights struct here? e.g. `Weights weights` if that works?

**Review by oraclown — COMMENTED** [review]
Looking good so far, @pawanbisht62 . Thanks. I also like Frank's suggestion about the storage item (if possible)

File: src/ParachainRegistry.sol (hunk 1 above)
Comment (pawanbisht62) [reply src/ParachainRegistry.sol:16]: yes it'll work, thanks for the suggestion.

**Review by oraclown — APPROVED** [review]
Thanks for the update, @pawanbisht62

**Review by evilrobot-01 — APPROVED** [review]
LGTM

**Comment by evilrobot-01** [comment]
@oraclown successfully tested a few calls manually, good to merge when youre ready.

**Pull request merged**